app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  def Show
+
+  end
+  def new
+    @post = Post.find_by_id(params[:post_id])
+  end
+
+  def create
+
+    p "here is there"
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params) do |comment|
+      comment.user_id=current_user[:id]
+    end
+    #debugger
+    if @comment.save
+      redirect_to post_path(@post)
+    end
```

Comment:
What will your code do if comment not save?

Reply:
Done